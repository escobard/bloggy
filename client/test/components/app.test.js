import React, { Component } from 'react'
import App from '../../src/components/app';
import renderer from 'react-test-renderer';

describe('App', () => {
  describe('snapshots', () => {
    it('renders one variant', () => {
      const tree = renderer.create(
        <App />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('renders another variant', () => {
      const tree = renderer.create(
        <App />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});

test('render a label', () => {
    const wrapper = shallow(
        <App />
    ).toJSON();
    expect(wrapper).toMatchSnapshot();
});