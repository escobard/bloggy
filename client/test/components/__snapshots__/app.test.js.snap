// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App - with actual state +++ render the connected(SMART) component 1`] = `undefined`;

exports[`Header - with actual state +++ render the connected(SMART) component 1`] = `undefined`;

exports[`Header - with material ui +++capturing Snapshot of Header 1`] = `undefined`;
